### ISSUE TITLE
Builder error

### ISSUE THREAD

**Issue by Cheater-Hub** [UnamSanctam/SilentCryptoMiner#1331]
![minererror](https://user-images.githubusercontent.com/107192004/208961270-0e36e886-6088-43d2-84d8-9002f8891f47.png)

**Comment by Cheater-Hub** [comment]
any idea what this is?

**Comment by masterjek** [comment]
If you read the text of the error, then the error indicates that there are spaces in the path to the final file. Most likely, you need to specify another path to create a miner, shorter and without spaces.

**Comment by UnamSanctam** [comment]
Yes it's as @masterjek says, change the path you're building in, the windres resource compiler does not support spaces in it's path. Right now you're also building inside the Temp folder which isn't very good.

**Comment by Cheater-Hub** [comment]
yes I didn't know what was meant by spaces but thanks anyway
